### PR TITLE
samples(healthcare v1): initial consent store CRUD method samples with new subdirectory

### PR DESCRIPTION
**Pull request by noerog**
New directory and initial samples for Healthcare API consent modality. Part of b/224797364.

## Checklist
- [X] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [X] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [X] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [X] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Comment by noerog** [comment]
All tests are failing due to permission errors. I am in the process of adding the required roles on our testing service account.

**Comment by dandhlee** [comment]
All necessary permissions have been added.

File: healthcare/api-client/v1/consent/README.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=healthcare/api-client/v1/consent/README.rst
+
+
+This directory contains samples for Cloud Healthcare API. `Cloud Healthcare API`_ implements healthcare-native protocols and formats to accelerate ingestion, storage, analysis, and integration of healthcare data with cloud-based applications.
+- See the `migration guide`_ for information about migrating to Python client library v0.25.1.
+
+.. _migration guide: https://cloud.google.com/vision/docs/python-client-migration
```
Comment (parthea) [inline healthcare/api-client/v1/consent/README.rst:13]: This migration guide is for the Cloud Vision client. This is referencing a `google-cloud` client from 2017. https://pypi.org/project/google-cloud/0.25.1/

File: healthcare/api-client/v1/consent/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+google-api-python-client==2.40.0
+google-auth-httplib2==0.1.0
+google-auth==2.5.0
+google-cloud==0.34.0
```
Comment (parthea) [inline healthcare/api-client/v1/consent/requirements.txt:4]: This is an old version from 2018: https://pypi.org/project/google-cloud/0.34.0/

**Review by parthea — CHANGES_REQUESTED** [review]

File: healthcare/api-client/v1/consent/requirements.txt (hunk 2 above)
Comment (noerog) [reply healthcare/api-client/v1/consent/requirements.txt:4]: The guidance from that version says to use the product-specific google-cloud-* packages, but the Healthcare API doesn't have one: https://github.com/googleapis/google-cloud-python

File: healthcare/api-client/v1/consent/README.rst (hunk 1 above)
Comment (noerog) [reply healthcare/api-client/v1/consent/README.rst:13]: Done.

**Comment by noerog** [comment]
Ping @loferris @parthea

File: healthcare/api-client/v1/consent/requirements.txt (hunk 2 above)
Comment (parthea) [reply healthcare/api-client/v1/consent/requirements.txt:4]: Can you help me locate the code in this sample that uses `google-cloud`? In other words, what error will we get if we remove `google-cloud` from this requirements file?

Comment (noerog) [reply healthcare/api-client/v1/consent/requirements.txt]: You're right, it looks like we don't need it. I thought nox would catch that.

**Review by parthea — APPROVED** [review]

**Pull request merged**